mixins/list.go

```diff
@@ -231,10 +244,16 @@ func (l *List) SetScrollOffset(scrollOffset int) {
 	}
 	s := l.outer.Size().Contract(l.outer.Padding())
 	if l.orientation.Horizontal() {
+		if l.scrollRound && l.itemSize.W != 0 {
+			scrollOffset = scrollOffset + scrollOffset%l.itemSize.W
```

Comment:
I tested this locally and triggered a crash.  I'm pretty sure that you need to subtract the remainder, not add it.  Also you can use `-=` and `+=` here.

So this line should probably be `scrollOffset -= scrollOffset%l.itemSize.W`, instead.

Reply:
Interesting what the reason of the crush, but you are right, I'll change this lines.